### PR TITLE
Added ruby messaging examples

### PR DESCRIPTION
**Pull request by jmulford-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `gitbook` 

- [ ] I made sure that the site looks great and functions perfectly live
- [ ] I ran splell check
- [ ] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [ ] I did these things
- [ ] I'm ready for these changes to be made live

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-785.s3-website-us-east-1.amazonaws.com/

File: messaging/methods/media/getMedia.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ var a = b;
 {% sample lang="ruby" %}
 
 ```ruby
-# Ruby Example
+downloaded_media = messaging_client.get_media("12345", "mediaId")
```
Comment (ghost) [inline messaging/methods/media/getMedia.md]: Can you show how to work with the download_media object? 

Saving to disk or pushing to AWS kinda deal

File: messaging/methods/media/uploadMedia.md [hunk 2]
```diff
@@ -50,7 +50,9 @@ var a = b;
 {% sample lang="ruby" %}
 
 ```ruby
-# Ruby Example
+f = File.open("some file", "r")
+file_content = f.read
+messaging_client.upload_media("12345", "mediaId", file_content.length.to_s, file_content, :content_type => "application/octet-stream", :cache_control => "no-cache")
```
Comment (ghost) [inline messaging/methods/media/uploadMedia.md]: Does this return anything? 

Do I get a new URL ?

**Review by ghost — CHANGES_REQUESTED** [review]
A few more examples on working with the media objects would be helpful.

Don't need to write the code to actually do the thing, just show how the object is structured

File: messaging/methods/media/uploadMedia.md (hunk 2 above)
Comment (jmulford-bw) [reply messaging/methods/media/uploadMedia.md]: No return according to the OAS

File: messaging/methods/media/getMedia.md (hunk 1 above)
Comment (jmulford-bw) [reply messaging/methods/media/getMedia.md]: Added writing of downloaded media to a file 👍

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-785.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-785.s3-website-us-east-1.amazonaws.com/

**Pull request merged**